### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by jelmerdemaat** [icetee/remote-ftp#91]
Auto-generated issue:

Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:

``` json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```

```
Package.getActivationCommands (C:\Users\jelmaa\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:790:9)
Package.hasActivationCommands (C:\Users\jelmaa\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:715:20)
<unknown> (C:\Users\jelmaa\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:169:24)
Package.measure (C:\Users\jelmaa\AppData\Local\atom\app-0.196.0\resources\app.asar\src\package.js:147:15)
```

**Comment by samijnih** [comment]
+1

**Comment by grenierdev** [comment]
Merged pull request #98
